scripts/select-java

```diff
@@ -1,27 +1,24 @@
 #!/bin/bash
 
 function select_java_debian() {
+    local java
+    local arch
+    arch=$(dpkg --print-architecture)
+
     for version in "11" "8"; do
-        pkg_name=openjdk-${version}-jre-headless
-        if ! dpkg-query --status ${pkg_name}; then
-            continue
-        fi
-        java=$(dpkg -L ${pkg_name} | grep '/java$')
-        if [ -n "$java" ]; then
+        java=/usr/lib/jvm/java-11-openjdk-${arch}/bin/java
```

Comment:
Shouldn't you replace `11` with `${version}` here? You are checking for java 11 in each iteration of the loop.

Reply:
@denesb hi Botond, thanks! fixed and repushed.